src/repository/screens/repository.screen.js

```diff
@@ -145,7 +166,7 @@ class Repository extends Component {
     const openIssues = pureIssues.filter(issue => issue.state === 'open');
     const closedIssues = pureIssues.filter(issue => issue.state === 'closed');
 
-    const repositoryActions = [starred ? '★ Unstar' : '★ Star'];
+    const repositoryActions = ['Share', starred ? 'Unstar' : 'Star'];
```

Comment:
Same, nothing extremely important however but I also tend to see it in the bottom.